RafRate.API/Configuration/Extentions/DataBaseExtension.cs

```diff
@@ -0,0 +1,11 @@
+﻿using RafRate.Data.Contexts;
+using Microsoft.EntityFrameworkCore;
+
+namespace RafRate.API.Configuration.Extentions;
+
+public static class DataBaseExtension
+{
+    public static IServiceCollection AddDataBase(this IServiceCollection services)
+    => services.AddDbContext<AppDbContext>(options => 
+        options.UseSqlServer(ConnectionStringContext.ConnectionString));
```

Comment:
And you should use Environment Variables to store your connection string, not your config file